src/ecl_ekf_analysis/log_processing/data_version_handler.py

```diff
@@ -157,3 +210,40 @@ def field_name_wo_brackets(field_name: str) -> str:
     if field_name.endswith(']'):
         return field_name[:-3]
     return field_name
+
+def get_innovation_message_and_field_name(
+        ulog: ULog, field_descriptor: str, topic: str = 'innovation') -> tuple([str, str]):
```

Comment:
change `tuple([str, str])` to `Tuple[str, str]` and add `from typing import Tuple` at top.